crates/ruma-federation-api/src/thirdparty/exchange_invite.rs

```diff
@@ -46,7 +54,9 @@ pub mod v1 {
         pub state_key: OwnedUserId,
 
         /// The content of the invite event.
-        pub content: ThirdPartyInvite,
+        ///
+        /// It must have a `membership` of `invite` and the `third_party_invite` field must be set.
+        pub content: RoomMemberEventContent,
```

Comment:
Maybe this should be `Raw<RoomMemberEventContent>`?

Reply:
However the fields are clearly defined in the spec, so we would probably gain nothing by keeping the raw JSON.